### PR TITLE
Guidelines

### PR DESCRIPTION
**Pull request by Akarys42**
This PR adds a guidelines section on the website.

The currently written guidelines include:
- Reporting a staff member
- Who can see a ModMail thread?
- Food policy
- Syncing bans from Discord to other platforms
- Who can use PluralKit?
- Creating seasonal branding
- Hosting a Minecraft server in Quilt
- Staff membership applications
- Navigating conflicts of interest
- Staff activity requirements

[Preview](https://preview-125.quiltmc-org.pages.dev/en/community/guidelines/)

---
See preview on Cloudflare Pages: https://preview-125.quiltmc-org.pages.dev

### CODE REVIEW

**Review by BucketOfCompasses — APPROVED** [review]

File: src/pages/en/community/guidelines.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+# Moderation
+
+<Message>
+	# Reporting a staff member
+
+	If you feel the need to report a member of staff, the right person to contact will depend on their roles:
+	- Developer: Contact ModMail or a Moderator directly
+	- Moderator: Contact a Manager or ModMail
+	- Manager: Contact a member of the Administrative Board or another Manager
+	- Administrative Board: Contact a Manager or another member of the Administrative Board
```
Comment (SilverAndro) [inline src/pages/en/community/guidelines.mdx:26]: I feel like this should probably also mention how to report a CT who isnt a mod?

File: src/pages/en/community/guidelines.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+	ModMail threads are, in most cases, only seen by Moderators and Community Managers. However, we may choose to share a thread with the Administrative Board on request,
+	for oversight purposes.
+
+	Application threads will also be seen by the Administrative Board and Community Team, as they are a part of the application process.
+
+	We may also choose to share a stripped down version of the thread with the Community Team, with any personal information redacted, if required as part of an
+	action against a member of the staff.
+
+	If we would like to share a thread further, we will ask for your permission first. Please note that bad faith threads are exempt from this rule.
```
Comment (SilverAndro) [inline src/pages/en/community/guidelines.mdx:40]: we have also once pulled the CT into an ban appeal thread as it concerned some of the members, would that be prohibited in this case, or require stripping it down &| asking permission?

File: src/pages/en/community/guidelines.mdx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+	To avoid stigmatizing food, conversations about food and drinks are allowed anywhere in our community space. We ask you to not request
+	food content warnings, but you are free to content warn your own content as you wish.
+
+	We recognize that this policy may not be ideal for everyone, but we believe it is the approach that is most inclusive of our community and inline with our values.
+</Message>
+
+<Message>
+	# Syncing bans from Discord to other platforms
+
+	We take a very firm approach to moderation on Discord, as it is a social space before anything else. However, we can take a more lax approach
```
Comment (SilverAndro) [inline src/pages/en/community/guidelines.mdx:55]: "can" or "do"?

File: src/pages/en/community/guidelines.mdx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+	when it comes to other platforms to not block users entirely from voicing their opinions on Quilt's development or get support.
+
+	As such, we will sync bans from Discord to other platforms under any of the following conditions:
+	- The activity of the banned user in another platform would make vulnerable users feel uncomfortable and unable to interact in that platform.
+	- The user has been banned for spamming or trolling.
+	- We do not want our projects to be associated with that user.
```
Comment (SilverAndro) [inline src/pages/en/community/guidelines.mdx:61]: This is so incredibly vague its basically useless and just turns the guidelines back into "well maybe we dont like you"

File: src/pages/en/community/guidelines.mdx [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+# Community
+
+<Message>
+	# Who can use PluralKit?
+
+	PluralKit is reserved for accessibility purpose. We allow any good faith usage of that tool under that condition.
+
+	For example:
+	- Allowed: Using PluralKit to distinguish between members of a plural system, endogenic or traumagenic, or questioning.
```
Comment (SilverAndro) [inline src/pages/en/community/guidelines.mdx:72]: I feel like this needs some sort of wording change but im not quite sure how to do it

I dislike the (what i read as) implication of only 3 possible "states" of plurality and a requirement to know your system origin, but it feels important to explicit about endogenic systems at least somewhere

**Review by SilverAndro — COMMENTED** [review]
some nitpicking, nothing too major tho

**Comment by gdude2002** [comment]
I've gone ahead and rewritten this document, fixing up some of the styling and removing the hard line-breaks.

One question: Why is "working in a professional capacity" listed as a conflict of interest? I don't really understand that one.

**Comment by gdude2002** [comment]
> One question: Why is "working in a professional capacity" listed as a conflict of interest? I don't really understand that one.

I've replaced this with a more general point about influence and power

File: src/pages/en/community/guidelines.mdx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+# Moderation
+
+<Message>
+	## Reporting a staff member
+
+	If you wish to report a member of staff, you should contact another staff member based on the roles of whomever you're making a report for:
+
+	- **Admin Board:** Contact a Community Manager or another member of the Admin Board
+	- **Community Manager:** Contact a member of the Admin Board or another Community Manager
+	- **Community Team:** Contact a member of the Admin Board or a Community Manager
```
Comment (Akarys42) [inline src/pages/en/community/guidelines.mdx:30]:
```suggestion
	- **Community Team:** Contact a Community Manager or a member of the Admin Board
```
Conflicts should be attempted to be solved internally first, if possible

**Review by LambdAurora — APPROVED** [review]

**Comment by woodiertexas** [comment]
In https://preview-125.quiltmc-org.pages.dev/en/community/guidelines/#staff-activity-requirements, shouldn't there be an entry for the Community Team's activity requirements?

**Comment by Akarys42** [comment]
> In https://preview-125.quiltmc-org.pages.dev/en/community/guidelines/#staff-activity-requirements, shouldn't there be an entry for the Community Team's activity requirements?

It was a conscious decision not to

**Comment by woodiertexas** [comment]
> > In https://preview-125.quiltmc-org.pages.dev/en/community/guidelines/#staff-activity-requirements, shouldn't there be an entry for the Community Team's activity requirements?
> 
> It was a conscious decision not to

oh ok

**Comment by Southpaw1496** [comment]
These guidelines will remain on the guidelines branch on the repository and we will open a new request when the team has time to review and update them if necessary.

**Comment by Akarys42** [comment]
I would argue this branch should be deleted.

**Pull request closed without merging**